magda-web-client/src/Components/Common/ContactPoint.js

```diff
@@ -27,27 +26,47 @@ class ContactPoint extends React.Component {
             <MagdaNamespacesConsumer ns={["datasetPage"]}>
```

Comment:
Do we still want to add this if we have the "{this.props.contactPoint ? (..." in line 31? We want to provide an alternate contact point text if the contact point doesn't exist, rather than remove text altogether.